prog/dftb+/lib_dftbplus/mainio.F90

```diff
@@ -3823,9 +3830,9 @@ subroutine printEnergies(energy, TS, electronicSolver, tDefinedFreeE)
     if (electronicSolver%providesEigenvals) then
       write(stdOut, format2U) "Extrapolated to 0", energy%Ezero, "H", Hartree__eV * energy%Ezero,&
           & "eV"
-      write(stdOut, format2U) "Total Mermin free energy", energy%EMermin, "H",&
-          & Hartree__eV * energy%EMermin, "eV"
     end if
+    write(stdOut, format2U) "Total Mermin free energy", energy%EMermin, "H",&
```

Comment:
Hier we could also make an `if (electronicSolver%providesFreeEnergy) then` block...